puppet-enterprise-metrics-collection.md

```diff
@@ -0,0 +1,46 @@
+# Puppet Enterprise Metrics Collection
+
+## Summary
+
+Puppet Enterprise users should install metrics collection. This standard defines how to gather, retain, and graph metrics for Puppet Enterprise components in a format that is easily shareable with Puppet Support.  
```

Comment:
Not sure that this first sentence makes much sense: `Puppet Enterprise users should install metrics collection.`

Are we trying to say that users should monitor PE metrics or that they should use the specific module we reference below?

Reply:
I tried to clean that up a bit.